app/views/welcome/_next-event.html.erb

```diff
@@ -18,16 +18,14 @@
     <% else %>
       <p>
         There are currently no presentations for this event.
-        <%= link_to 'Sign up', new_presentation_path %>
+        <%= render 'next-event-links' %>
```

Comment:
Maybe we could rename this partial to `new_presentation_link` or something like that? That way we could use it in other areas of the system, too.